.github/workflows/gating.yaml

```diff
@@ -122,3 +122,31 @@ jobs:
         run: |
           podman  pull ${{ env.IMAGE_REGISTRY_QUAY }}:${{ env.IMAGE_TAG }}
           podman  images
+
+      - name: Checkout sssd repository
+        uses: actions/checkout@v3
+        with:
+          path: sssd
```

Comment:
I am afraid you miss a repository here:
```yaml
  with:
    repository: my-org/my-tools
    path: my-tools
```
in case of sssd repo I think it can be:
```
  with:
    repository: SSSD/sssd-ci-containers
    path: sssd-ci-containers
```